newbank/server/NewBank.java

```diff
@@ -155,9 +154,12 @@ private String createLoginAccount(String request) {
 			String output = "The username already exists.\nPlease enter a unique username or type 'esc' to return to the menu screen.";
 			return output;
 		} else {
-			Customer newCustomer = new Customer(customerName, userName, password, iban);       // create new customer
+			Customer newCustomer = new Customer(customerName, userName, password);       // create new customer
 			newCustomer.addAccount(new Account("Current Account", "Main", 00.0));    // create a default account for the customer
 			bank.customers.put(userName, newCustomer);        // add the customer to the list of customers and assign their username
+			// generate iban for new customer who has just been created and return in the output
+			Account MainAccount = newCustomer.getAccount("Main");
+			String iban = MainAccount.getIBAN();
 			String output = "New user '" + userName + "' created.\n" +
 					"We also created an initial bank account:'Main' with the IBAN: " + iban + " for you.\n" +
 					"Please Download the Google Authenticator App and use the key NY4A5CPJZ46LXZCP to set up your 2FA.";
```

Comment:
```suggestion
					"Please Download the Google Authenticator App and use the key NY4A5CPJZ46LXZCP to set up your 2FA.";
```
can we remove this bit please as functionality doesn't exist yet